src/view/components/common/iframe-legacy/Iframe-legacy-component.ts

```diff
@@ -91,4 +102,17 @@ export class IframeLegacyComponent extends SiteChangeListener {
             }
         }
     }
+
+    /**
+     * Force to reload the current iframe component portlet,
+     * with the specified portlet url
+     * @param url Url of the portlet to display
+     */
+    reloadIframePortlet(url: string): void {
+        if(url !== undefined && url !== '') {
+            let urlSplit = url.split('/');
+            let id = urlSplit[urlSplit.length - 1];
+            this.iframe = this.loadURL(this.routingService.getPortletURL(id) + '&in_frame=true&frame=detailFrame');
+        }
```

Comment:
Why don't we just do the following?

this.iframe = this.loadURL(url + '&in_frame=true&frame=detailFrame');